src/backend/XmlHttpRequest.ts

```diff
@@ -69,11 +75,12 @@ export default class XmlHttpRequest extends BaseFileSystem implements FileSystem
     if (typeof(opts.index) === "string") {
       XmlHttpRequest.FromURL(opts.index, cb, opts.baseUrl, false);
     } else {
-      cb(null, new XmlHttpRequest(opts.index, opts.baseUrl, false));
+      cb(null, new XmlHttpRequest(opts.index, opts.baseUrl, false, opts.use));
```

Comment:
The constructor is private in V2.0 and users should not call it directly. `listingUrlOrObj` as a string in the constructor is deprecated and is removed in V2.0. `Create` handles that variant like it does in V1.x -- asynchronously.

So, don't worry about that exception. This is one of the motivations for the `Create` method -- keeping constructors simple, and centralizing the checking of user-passed options!